src/processes/Processes.Library/Processes.Library.csproj

```diff
@@ -35,6 +35,7 @@
 
     <ItemGroup>
       <PackageReference Include="Flurl.Http.Signed" Version="3.2.4" />
+      <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
why those references? the project seems not to contain anything related to Flur or Json (and we should use System.Text.Json, not Newtonsoft anyway)

Reply:
you're right, removed that